decryptor/postgresql/pg_decryptor.go

```diff
@@ -376,7 +382,11 @@ func (proxy *PgProxy) handleQueryPacket(ctx context.Context, packet *PacketHandl
 }
 
 func (proxy *PgProxy) handleBindPacket(ctx context.Context, packet *PacketHandler, logger *log.Entry) (bool, error) {
-	bind := proxy.protocolState.PendingBind()
+	bind, err := proxy.protocolState.LastBind()
```

Comment:
As I see `LastBind()` could return nil packet with nil error. Just make sure it's a safe operation.

Reply:
added nil checks